src/sagemaker/jumpstart/model.py

```diff
@@ -786,6 +791,70 @@ def register_deploy_wrapper(*args, **kwargs):
 
         return model_package
 
+    def display_benchmark_metrics(self):
+        """Display Benchmark Metrics for deployment configs."""
+        config_names = []
+        latencies = []
+        through_puts = []
+        instance_types = []
+        instance_rates = []
```

Comment:
let's separate out the fields into two category:
1. metadata: config-name/id, instance type, instance rate, currently selected
2. metrics: could be anything with schema of `{"name", str, "value": any: "unit": str}`